samples/client/petstore/kotlin-model-prefix-type-mappings/src/main/kotlin/org/openapitools/client/models/ApiUserOrPetOrArrayString.kt

```diff
@@ -147,4 +147,58 @@ data class ApiUserOrPetOrArrayString(var actualInstance: Any? = null) {
             }.nullSafe() as TypeAdapter<T>
         }
     }
+
+    companion object {
+        /**
+        * Validates the JSON Element and throws an exception if issues found
+        *
+        * @param jsonElement JSON Element
+        * @throws IOException if the JSON Element is invalid with respect to ApiUserOrPetOrArrayString
+        */
+        @Throws(IOException::class)
+        fun validateJsonElement(jsonElement: JsonElement?) {
```

Comment:
@CaptainAye Thanks for this PR. 
I have one question, why is this jsonElement optional?
If a jsonElement is optional is invalid, or am I wrong?
If we really need to receive the jsonElement as optional, I think we should check if it's null and throw an exception as early as possible, to avoid the !! and required in this code, that can cause a runtime exception.

Reply:
I kept `jsonElement` optional in the definition to maintain compatibility with `validateJsonElement()` method in `src/main/resources/kotlin-client/data_class.mustache`. Idd it seems that the jsonElement could be just required in all of these places. I wanted to avoid broadening the scope of this pr.
I pushed the change to check that jsonElement isn't at the beginning of the method. 
Let me know if you think that's good enough.

Reply:
Looks good to me 👍